b2/_internal/console_tool.py

```diff
@@ -4624,6 +4624,56 @@ def _run(self, args):
         return 0
 
 
+class Key(Command):
+    """
+    Application keys management subcommands.
+
+    For more information on each subcommand, use ``{NAME} key SUBCOMMAND --help``.
+
+    Examples:
+
+    .. code-block::
+
+        {NAME} key list
+        {NAME} key create my-key listFiles,deleteFiles
+        {NAME} key remove 005c398ac3212400000000010
```

Comment:
```suggestion
        {NAME} key remove 005c398ac3212400000000010
```
there is no `key remove` command
```suggestion
        {NAME} key delete 005c398ac3212400000000010
```

Reply:
bump